proxy_map.go

```diff
@@ -950,7 +1052,39 @@ func (m *Map) addIndex(ctx context.Context, indexConfig types.IndexConfig) error
 	return err
 }
 
-func (m *Map) addEntryListener(ctx context.Context, flags int32, includeValue bool, key interface{}, predicate predicate.Predicate, handler EntryNotifiedHandler) (types.UUID, error) {
+//func (m *Map) addListener(ctx context.Context, flags int32, includeValue bool, key *interface{}, predicate *predicate.Predicate, listener MapListener, methodName string) (types.UUID, error) {
+//	var err error
+//	var keyData serialization.Data
+//	var predicateData serialization.Data
+//	if key != nil {
+//		if keyData, err = m.validateAndSerialize(*key); err != nil {
+//			return types.UUID{}, err
+//		}
+//	}
+//	if predicate != nil {
+//		if predicateData, err = m.validateAndSerialize(*predicate); err != nil {
+//			return types.UUID{}, err
+//		}
+//	}
+//	subscriptionID := types.NewUUID()
+//	addRequest := m.makeListenerRequest(keyData, predicateData, flags, includeValue)
+//	listenerHandler := func(msg *proto.ClientMessage) {
+//		m.makeListenerDecoder(msg, keyData, predicateData, m.mapListenerEventHandler(listener, methodName))
+//	}
+//	removeRequest := codec.EncodeMapRemoveEntryListenerRequest(m.name, subscriptionID)
+//	err = m.listenerBinder.Add(ctx, subscriptionID, addRequest, removeRequest, listenerHandler)
+//	return subscriptionID, err
+//}
+
+func (m *Map) addEntryListener(
+	ctx context.Context,
+	flags int32,
+	includeValue bool,
+	key interface{},
+	predicate predicate.Predicate,
+	handler EntryNotifiedHandler,
+	methodName string,
```

Comment:
I don't like having to pass `methodName` just to put it in an error message. How about removing that and changing the error logging code to something like:
```
p.logger.Errorf("preparing entry notified event: %w", err)
```

Reply:
Changed it in fa35423